Backend_NodeJS/src/config/index.js

```diff
@@ -4,9 +4,8 @@ const { PORT } = process.env;
 const port = PORT || 8080;
 const dbUri =
   // "mongodb+srv://admin:admin@revive.p6okv5v.mongodb.net/?retryWrites=true&w=majority";
-  "mongodb+srv://admin:admin@revivegallery.9qab75b.mongodb.net/?retryWrites=true&w=majority"
+  "mongodb+srv://admin:admin@projectgallery.rbwfrho.mongodb.net/?retryWrites=true&w=majority"
```

Comment:
@SriHarikaV It's better to not push this config file, these are local to everyone. So we do not need these changes in git repo.

Reply:
I will make sure not to add this file next time.

Reply:
@SriHarikaV Sounds good. Thank you. I am approving it now.